routes/api/users.js

```diff
@@ -45,6 +45,10 @@ router.post('/profiles/:username/follow', checkToken, userControllers.follow);
 
 // Un-follow
 router.delete('/profiles/:username/follow', checkToken, userControllers.unFollow);
+
+router.get('/profile/:username/followers', checkToken, userControllers.getFollowers);
```

Comment:
@shemaeric On the route, you are using **checkToken** which makes it mandatory for someone to be logged in to view your followers or the people you follow. Since the profile page is accessible to everyone, those two endpoints should not be authenticated.